[Build Failure] Basic UI failed to build, depends on python version

Running the ESH build on Jenkins resulted with build failure : 

```
mvn clean install 

....

[INFO] ------------------------------------------------------------------------
[INFO] Building Eclipse SmartHome Basic UI 0.10.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 

.....

[INFO] --- frontend-maven-plugin:1.0:install-node-and-npm (Install node and npm) @ org.eclipse.smarthome.ui.basic ---
[INFO] Node v5.4.0 is already installed.
[INFO] NPM 3.3.12 is already installed.
[INFO] 
[INFO] --- frontend-maven-plugin:1.0:npm (npm install) @ org.eclipse.smarthome.ui.basic ---
[INFO] Running 'npm install' in /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic
[WARNING] npm WARN prefer global node-gyp@3.6.2 should be installed with -g
[INFO] 
[INFO] > node-sass@4.7.2 install /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-sass
[INFO] > node scripts/install.js
[INFO] 
[INFO] Cached binary found at /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/npm_cache/node-sass/4.7.2/linux-x64-47_binding.node
[INFO] 
[INFO] > node-sass@4.7.2 postinstall /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-sass
[INFO] > node scripts/build.js
[INFO] 
[INFO] Binary found at /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-sass/vendor/linux-x64-47/binding.node
[INFO] Testing binary
[INFO] Binary has a problem: [Error: /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-sass/vendor/linux-x64-47/binding.node: ELF file OS ABI invalid]
[INFO] Building the binary locally
[INFO] Building: /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node/node /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-gyp/bin/node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
[ERROR] gyp info it worked if it ends with ok
[ERROR] gyp verb cli [ '/var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node/node',
[ERROR] gyp verb cli   '/var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-gyp/bin/node-gyp.js',
[ERROR] gyp verb cli   'rebuild',
[ERROR] gyp verb cli   '--verbose',
[ERROR] gyp verb cli   '--libsass_ext=',
[ERROR] gyp verb cli   '--libsass_cflags=',
[ERROR] gyp verb cli   '--libsass_ldflags=',
[ERROR] gyp verb cli   '--libsass_library=' ]
[ERROR] gyp info using node-gyp@3.6.2
[ERROR] gyp info using node@5.4.0 | linux | x64
[ERROR] gyp verb command rebuild []
[ERROR] gyp verb command clean []
[ERROR] gyp verb clean removing "build" directory
[ERROR] gyp verb command configure []
[ERROR] gyp verb check python checking for Python executable "python2" in the PATH
[ERROR] gyp verb `which` succeeded python2 /usr/bin/python2
[ERROR] gyp verb check python version `/usr/bin/python2 -c "import platform; print(platform.python_version());"` returned: "2.4.3\n"
[ERROR] gyp ERR! configure error 
[ERROR] gyp ERR! stack Error: Python executable "/usr/bin/python2" is v2.4.3, which is not supported by gyp.
[ERROR] gyp ERR! stack You can pass the --python switch to point to Python >= v2.5.0 & < 3.0.0.
[ERROR] gyp ERR! stack     at Object.failPythonVersion (/var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-gyp/lib/configure.js:492:19)
[ERROR] gyp ERR! stack     at Object.<anonymous> (/var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-gyp/lib/configure.js:474:14)
[ERROR] gyp ERR! stack     at ChildProcess.exithandler (child_process.js:204:7)
[ERROR] gyp ERR! stack     at emitTwo (events.js:87:13)
[ERROR] gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
[ERROR] gyp ERR! stack     at maybeClose (internal/child_process.js:821:16)
[ERROR] gyp ERR! stack     at Socket.<anonymous> (internal/child_process.js:319:11)
[ERROR] gyp ERR! stack     at emitOne (events.js:77:13)
[ERROR] gyp ERR! stack     at Socket.emit (events.js:169:7)
[ERROR] gyp ERR! stack     at Pipe._onclose (net.js:469:12)
[ERROR] gyp ERR! System Linux 2.6.18-194.3.1.el5xen
[ERROR] gyp ERR! command "/var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node/node" "/var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
[ERROR] gyp ERR! cwd /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node_modules/node-sass
[ERROR] gyp ERR! node -v v5.4.0
[ERROR] gyp ERR! node-gyp -v v3.6.2
[ERROR] gyp ERR! not ok 
[ERROR] Build failed with error code: 1
[WARNING] npm WARN EPEERINVALID ajv-keywords@3.1.0 requires a peer of ajv@^6.0.0 but none was installed.
[WARNING] npm WARN EPACKAGEJSON org.eclipse.smarthome.ui.basic@0.10.0 No repository field.
[WARNING] npm WARN EPACKAGEJSON org.eclipse.smarthome.ui.basic@0.10.0 No license field.
[ERROR] npm ERR! Linux 2.6.18-194.3.1.el5xen
[ERROR] npm ERR! argv "/var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node/node" "/var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/node/node_modules/npm/bin/npm-cli.js" "install"
[ERROR] npm ERR! node v5.4.0
[ERROR] npm ERR! npm  v3.3.12
[ERROR] npm ERR! code ELIFECYCLE
[ERROR] 
[ERROR] npm ERR! node-sass@4.7.2 postinstall: `node scripts/build.js`
[ERROR] npm ERR! Exit status 1
[ERROR] npm ERR! 
[ERROR] npm ERR! Failed at the node-sass@4.7.2 postinstall script 'node scripts/build.js'.
[ERROR] npm ERR! Make sure you have the latest version of node.js and npm installed.
[ERROR] npm ERR! If you do, this is most likely a problem with the node-sass package,
[ERROR] npm ERR! not with npm itself.
[ERROR] npm ERR! Tell the author that this fails on your system:
[ERROR] npm ERR!     node scripts/build.js
[ERROR] npm ERR! You can get their info via:
[ERROR] npm ERR!     npm owner ls node-sass
[ERROR] npm ERR! There is likely additional logging output above.
[ERROR] 
[ERROR] npm ERR! Please include the following file with any support request:
[ERROR] npm ERR!     /var/lib/jenkins/jobs/Test-Job/workspace/extensions/ui/org.eclipse.smarthome.ui.basic/npm-debug.log

...

[INFO] Eclipse SmartHome UI Extensions .................... SUCCESS [  0.045 s]
[INFO] Eclipse SmartHome Icon Sets ........................ SUCCESS [  0.059 s]
[INFO] Eclipse SmartHome Classic Icon Set ................. SUCCESS [  4.243 s]
[INFO] Eclipse SmartHome Basic UI ......................... FAILURE [01:06 min]
[INFO] Eclipse SmartHome Classic UI ....................... SKIPPED
[INFO] Eclipse SmartHome Paper UI ......................... SKIPPED
[INFO] Eclipse SmartHome Voice Services ................... SKIPPED

```

@kaikreuzer, do you know who can help with this issue ?

@svilenvul Did you see the error message regarding the python version?

> [ERROR] gyp verb check python version `/usr/bin/python2 -c "import platform; print(platform.python_version());"` returned: "2.4.3\n"
[ERROR] gyp ERR! configure error 
[ERROR] gyp ERR! stack Error: Python executable "/usr/bin/python2" is v2.4.3, which is not supported by gyp.
[ERROR] gyp ERR! stack You can pass the --python switch to point to Python >= v2.5.0 & < 3.0.0.

If you have a newer python then 2.4.3 installed, maybe it is not on $PATH so npm cannot find it?

--- a/docs/documentation/development/ide.md
+++ b/docs/documentation/development/ide.md
@@ -11,6 +11,7 @@ Please ensure that you have the following prerequisites installed on your machin
 1. [Git](https://git-scm.com/downloads)
 1. [Maven 3.x](https://maven.apache.org/download.cgi)
 1. [Oracle JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+1. [Python >=2.5](https://www.python.org/downloads/) (to build UIs with npm)
 
 The Eclipse IDE is used for Eclipse SmartHome developments. The Eclipse Installer automatically prepares the IDE so that it comes with all required plug-ins, the correct workspace encoding settings, pre-configured code formatters and more. Simply follow these steps:
 

